src/lib/helpers.ts

```diff
@@ -247,3 +248,17 @@ export function getPrivateKey(input: string): string {
   if (!isHexString(privateKey, 32)) throw new Error(`Value ${input} not a valid private key`);
   return privateKey;
 }
+// Calculate the maximum block number to target with bundles by chainId.
+// In mainnet this is always the targetBlock, but in Goerli we add 24 blocks to the targetBlock.
+export function getMaxBlockByChainId(chainId: number, targetBlock: number) {
```

Comment:
id suggest doign some kind of mapping here, even though we arent supporting more than 2, its just easier to understand

```
  const additionalBlocks = {
   [MAINNET_CHAINID]:0
    [GOERLI_CHAIN_ID]:24
 }
```
then have your function just reference into the object to calc the offset, throw an error if an unsupported chain is used

Reply:
agree, and also better move all these chain ids and their additional target block values to constants leaving only the logic here